src/block-components/alignment/editor.scss

```diff
@@ -1,9 +1,48 @@
 .stk--column-flex,
+.stk--align-last-block-to-bottom,
+// for firefox :has polyfill
+.stk--alignment-polyfill > :is(.stk-inner-blocks, .stk-block-content):not(.stk--column-flex) {
```

Comment:
The two sets of CSS seem identical, I think you can just add the Firefox polyfill selector to the original CSS we had:

```css
.stk--column-flex,
// If an direct inner block has a top/bottom margin-auto, then we need to set
// the height to full (similar to .stk--column-flex) so the margin auto would
// look like it took effect..
:is(.stk-inner-blocks, .stk-block-content):not(.stk--column-flex):has(> .block-editor-inner-blocks > .block-editor-block-list__layout > .wp-block > :is(.stk--block-margin-top-auto, .stk--block-margin-bottom-auto)),
.stk--align-last-block-to-bottom,
// for firefox :has polyfill
.stk--alignment-polyfill > :is(.stk-inner-blocks, .stk-block-content):not(.stk--column-flex) {
	height: auto; // Reverse the 100% height from style.scss
	// ...other css
}
```